packages/react-dom/src/server/ReactPartialRendererHooks.js

```diff
@@ -202,24 +202,21 @@ export function finishHooks(
 
     children = Component(props, refOrContext);
   }
-  currentlyRenderingComponent = null;
```

Comment:
Did we lose currentlyRenderingComponent?

Reply:
Oops, yes -- added that in as well.